client/ayon_houdini/plugins/publish/validate_mkpaths_toggled.py

```diff
@@ -30,6 +30,6 @@ def get_invalid(cls, instance):
         for node in instance[:]:
             if node.parm("mkpath").eval() != 1:
                 cls.log.error("Invalid settings found on `%s`" % node.path())
-                result.append(node.path())
+                result.append(node)
```

Comment:
Making this change does mean that the validation error itself will now present itself with a less readable node path. So maybe in the raised `PublishValidationError` we also want to make sure we still format using the rop's path?